.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+version: 2
+jobs:
+  check_go_mod:
+    docker:
+      - image: golang:1.13
+    steps:
+      - checkout
+      - run:
+          name: "Check go module up to date"
+          command: ./scripts/ci/check_go_modules.sh
+
+  check_readme_sync:
+    docker:
+      - image: golang:1.13
+    steps:
+      - checkout
+      - run:
+          name: "Check README in sync"
+          command: ./scripts/ci/check_readme_in_sync.sh
+
+  build:
+    machine:
+      enabled: true
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - run:
+          name: "Install kind"
+          command: |
+            curl -L https://github.com/kubernetes-sigs/kind/releases/download/v0.5.1/kind-linux-amd64 --output kind
+            chmod +x ./kind
+            sudo mv ./kind /usr/local/bin
+      - run:
+          name: "Create kind cluster"
+          command: |
+            kind create cluster --config kind-config.yaml
+            echo 'export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"' >> $BASH_ENV
+      - run:
+          name: "Build Sonobuoy container image and add to kind cluster"
+          command: make container deploy_kind
+      - run:
+          name: "Run unit and stress tests"
+          command: VERBOSE=true make test stress
+      - run:
+          name: "Run Sonobuoy integration tests"
+          command: |
+            ./scripts/run_integration_tests.sh
+      - run:
+          name: "Start Sonobuoy run, wait, and gather results"
+          command: |
+            ./sonobuoy run --image-pull-policy=IfNotPresent -m Quick --wait
+            tarball=$(./sonobuoy retrieve)
+            ./scripts/ci/check_e2e_tarball.sh $tarball
+            mv $tarball /tmp/sonobuoy-tarball.tar.gz
+      - store_artifacts:
+          path: /tmp/sonobuoy-tarball.tar.gz
```

Comment:
Are you seeing the artifacts actually get uploaded? I dont see where to view them at
![image](https://user-images.githubusercontent.com/10273533/66484172-1e771900-ea6c-11e9-9671-e7df7abb9223.png)

Reply:
Still curious about this, but I just opted-in to the new UI on CircleCI and the artifacts were visible:
![image](https://user-images.githubusercontent.com/10273533/66484498-b4ab3f00-ea6c-11e9-8892-ad3bd9143bed.png)

Reply:
Are you looking at the `build` job? The artifacts are only available within that one job. It's not very intuitive but they are being uploaded: https://circleci.com/gh/vmware-tanzu/sonobuoy/95#artifacts/containers/0

I don't know if there's a way to see all the artifacts from the workflow overview. I can take a look into that to see if there's a way to make that a bit clearer.

Reply:
I used the `store_artifacts` step. I guess I should use `store_test_results` instead: https://circleci.com/docs/2.0/configuration-reference/#store_test_results

Reply:
Never mind, that seems to only be for specific test formats (JUnit, Cucumber etc)

Reply:
Still dont see them there for some reason. /shrug I'd think it was a permission issue but on the new view I can see them so it probably isnt that.

Devs are the only ones that really need to see them so if we both have _some_ way of seeing it its fine with me. Wanted to make sure they were getting uploaded was all. I'll just use the updated UI